custom_components/zwave_mqtt/sensor.py

```diff
@@ -86,3 +98,94 @@ class ZWaveBatterySensor(ZWaveSensor):
     def device_class(self):
         """Return the device class of the sensor."""
         return DEVICE_CLASS_BATTERY
+
+
+class ZWaveNodeStatisticsSensor(Entity):
+    """Representation of a Z-Wave node status sensor."""
+
+    def __init__(self, node):
+        """Initilize a generic Z-Wave device entity."""
+        self.node = node
+        self.options = node.options
+        self.attributes = {}
+        self.statistics = {}
+
+    @callback
+    def stats_changed(self, stats):
+        """Call when the statistics are changed."""
+        self.statistics = stats
+        stat_attributes = [
```

Comment:
Is this something anyone would ever automate on, or is this something that we could also expose via the websocket API? Seems like it would cause a lot of state changes.

Reply:
I've heard of a few people doing automations based on node RTT times and different stats but it does create a lot of state changes that probably 98% of zwave users aren't using.

Perhaps this is a good time to retire some of these attributes...

Reply:
Let's drop them and wait for people requesting them back, and seeing what the use cases are.

Reply:
Sorry to chimb in this discussion. Currently diving into this implementation to help you guys out where I can. I agree that some stuff probably needs to be retired. Some stats get updated really frequently. I'm also curious about the use case for doing stuff on RTT times. Most probably a workaround for a missing use case in old tooling, bad implementation or maybe a very rare edge case ;-)